react/src/types/projects.ts

```diff
@@ -0,0 +1,17 @@
+export interface Project {
+  description: string;
+  id?: number;
+  name: string;
+  ds_id?: string;
```

Comment:
nothing to change but just wanted to point out that this isn't a response from the backend but an extra property which was used in the angular client.  no need to change it but just an fyi.

Reply:
I can take this out with the other unneeded properties